### PR TITLE
Add type coverage

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #830

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -20,3 +20,8 @@ parameters:
     - Classes
     - Configuration
     - Tests
+
+  type_coverage:
+      return_type: 50
+      param_type: 30
+      property_type: 70
```
Comment (oliverklee) [inline phpstan.neon:27]: Let's go a lot higher!

**Comment by lukaszuznanski** [comment]
Damn, I clicked on update branch and merged main into branch... sorry, let's check if there is way to disable this button, or change it's behavior. I will fix this error in functional tests.

**Comment by oliverklee** [comment]
I've fixed this now:

```bash
git switch Feature/Add-type-coverage
git reset --hard a14f676
git rebase
git push -f
```

**Comment by lukaszuznanski** [comment]
I changed it to 100, 100 and 95 for properties. We can update it to 100, but first we need to add property type. This should work for now. Let's do it in another PR.

**Review by oliverklee — APPROVED** [review]

**Review by oliverklee — CHANGES_REQUESTED** [review]
Looks good. It should get an entry in the changelog, though.

**Review by oliverklee — APPROVED** [review]

**Pull request merged**